nix/default.nix

```diff
@@ -46,129 +45,131 @@
   enableNvidiaPatches ? false,
   nvidiaPatches ? false,
   hidpiXWayland ? false,
-}:
-assert lib.assertMsg (!nvidiaPatches) "The option `nvidiaPatches` has been removed.";
-assert lib.assertMsg (!enableNvidiaPatches) "The option `enableNvidiaPatches` has been removed.";
-assert lib.assertMsg (!hidpiXWayland) "The option `hidpiXWayland` has been removed. Please refer https://wiki.hyprland.org/Configuring/XWayland";
-  stdenv.mkDerivation {
-    pname = "hyprland${lib.optionalString debug "-debug"}";
-    inherit version;
+}: let
+  adapters = lib.concatLists [
```

Comment:
Just use ++ please

Reply:
Why

Reply:
Because this is dumb 🙄

Reply:
Ok, happy?

Reply:
Very

Reply:
I'm a huge fan of concatLists. Please change it again :trollface:

Reply:
I am not a huge fan of concatLists. It provides no real benefits than those can be achieved by ++. 
concatStrings or concatStringsSep is [another story](https://github.com/JohnRTitor/nix-conf/blob/33eabd65aeb88c38640cb40e856ad609c2fc426c/system/browsers.nix#L28) though.

Reply:
@NotAShelf bet

Reply:
@JohnRTitor the benefit is improved readability, as seen with `buildInputs`. It's not a big deal here but at least we maintain consistency.